frame/bags-list/src/list/tests.rs

```diff
@@ -359,7 +359,10 @@ mod list {
 		// make sure there are no duplicates.
 		ExtBuilder::default().build_and_execute_no_post_check(|| {
 			Bag::<Runtime>::get(10).unwrap().insert_unchecked(2, 10);
-			assert_eq!(List::<Runtime>::do_try_state(), Err("duplicate identified"));
+			assert_eq!(
```

Comment:
There is an `assert_err`, but both is fine.

Reply:
`assert_eq` was originally used so I just left it as it was.